sample_apps/simple_app_under_cca/temporary_instructions.txt

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
This file is copy-cloned from similar file for `simple_app_under_keystone/temporary_instructions.txt`. That one does run as a stand-alone shell script. And so should this one, but I have not tested this one specifically.

All these steps have been integrated carefully into `run_example.sh`, so, for now, I will just leave this file as-is.